### PR TITLE
FEATURE: dump blueprints to yaml

### PR DESCRIPTION
**Pull request by zaro0508**
Add the ability to dump stacker blueprints (in troposphere) to yaml format.

### CODE REVIEW

File: docs/config.rst [hunk 1]
```diff
@@ -44,6 +44,13 @@ See the `CloudFormation API Reference`_ for allowed stack name characters
 
 .. _`CloudFormation API Reference`: http://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateStack.html
 
+Dump Format
```
Comment (phobologic) [inline docs/config.rst]: Hmm - I think this makes more sense as a command line flag/argument to the `dump` command.  That way it's easy to dump in either format without having to manipulate configuration.  Any chance you could take a stab at that method?

Comment (zaro0508) [reply docs/config.rst]: My thought on this is that most people like to look at either json or yml but not really ever want to switch between the two that often.  Therefore i'm not sure how useful a command line switch would be.  This is the kind of thing that I like to set and never change again.

Comment (zaro0508) [reply docs/config.rst]: I just realized that it's already possible to override the dump format from a command line switch.

Add `dump_format: ${format}` to stacker.yml

On command line: `stacker build --region us-east-1 -e format=yaml -d DUMP conf/dev.env stacker.yaml`

Comment (phobologic) [reply docs/config.rst]: Sorry @zaro0508 - thought I'd responded to this in Slack, but looking back I realized I hadn't.  At this point I really think there are two best paths:

- Make it an optional argument to `build --dump`, like `--dump-format`.  The reason being that it's dump specific, and we don't really have any action specific configuration variables at the top level that I'm aware of.  It also feels very situational - each user would probably want to control this themselves.
- Make it a more generic configuration variable, like `template_format` - but then it'd need to be used in more places, like in the build action when uploading, etc.

I'm open to either of those - what do you think?

**Pull request closed without merging**